libbeat/docs/outputconfig.asciidoc

```diff
@@ -1186,3 +1211,32 @@ The following elliptic curve types are available:
 * P-256
 * P-384
 * P-521
+
+[[configuration-output-codec]]
+=== Output Codec Configuration
+
+For outputs that do not require a specific encoding, you can change the encoding
+by using the codec configuration. You can specify either the `json` or `format`
+codec. By default the `json` codec is used.
+
+*`json.pretty`*: If `pretty` is set to true, events will be nicely formatted. The default is false.
+
+Example configuration that uses the `json` codec with pretty printing enabled to write events to the console:
+
+[source,yaml]
+------------------------------------------------------------------------------
+output.console:
```

Comment:
Will the old `output.console.pretty: true` still work? If yes, we should deprecate it.